core-java-modules/core-java-arrays-guides/src/test/java/com/baeldung/print2DArrays/Print2DArraysTest.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.print2DArrays;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+
+import uk.org.webcompere.systemstubs.SystemOut;
+import uk.org.webcompere.systemstubs.annotations.SystemStub;
+
+public class Print2DArrayTest {
+    @SystemStub
+    private SystemOut systemOut;
+    
+    @Test
+    void whenPrint2D_thenUseNested() {
```

Comment:
Please review all of the test names. The BDD style of test naming has either 3 parts given/when/then, or two parts when/then.

With the two part style, the "when" is supposed to represent the action under test. In this test, the action under test is "print 2d array using nested loops". The "then" is "lines are output to system out".

So this test name should be something like: whenPrint2dArrayUsingNestedLoop_thenLinesAreWrittenToSystemOut

Also note the naming conventions for method names, lower camel case:
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names

Reply:
modified